contracts/GenArt721CoreV3.sol

```diff
@@ -20,13 +20,19 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     uint256 constant FOUR_WEEKS_IN_SECONDS = 2_419_200;
 
     // generic platform event fields
-    bytes32 constant FIELD_ARTBLOCKS_ADDRESS = "artblocksAddress";
+    bytes32 constant FIELD_ARTBLOCKS_PRIMARY_SALES_ADDRESS =
+        "artblocksPrimarySalesAddress";
+    bytes32 constant FIELD_ARTBLOCKS_SECONDARY_SALES_ADDRESS =
+        "artblocksSecondarySalesAddress";
     bytes32 constant FIELD_RANDOMIZER_ADDRESS = "randomizerAddress";
     bytes32 constant FIELD_ARTBLOCKS_CURATION_REGISTRY_ADDRESS =
         "curationRegistryAddress";
     bytes32 constant FIELD_ARTBLOCKS_DEPENDENCY_REGISTRY_ADDRESS =
         "dependencyRegistryAddress";
-    bytes32 constant FIELD_ARTBLOCKS_PERCENTAGE = "artblocksPercentage";
+    bytes32 constant FIELD_ARTBLOCKS_PRIMARY_SALES_PERCENTAGE =
+        "artblocksPrimaryPercentage";
+    bytes32 constant FIELD_ARTBLOCKS_SECONDARY_SALES_BPS =
```

Comment:
Tiny nit: should we use either Percentage or BPS just for consistency btwn the primary & secondary fields?

Reply:
Oh I see, so there's a decimal difference between % and BPS. In that case the naming change wouldn't make sense here, nvm! But I'm curious why there's a need for BPS vs. % decimals and if this is worth choosing one over the other for V3 and standardizing?

Reply:
hmm, yes, I've stuck with percentage everywhere internally, except had to use BPS since our typical AB royalties are defined as 2.5%, which is not able to be expressed as a uint. Externally, when we support the royalty registry, the `getRoyalties` function converts everything to BPS.

I think my current implementation logic here is something like:
 - let's keep the numbers percentage where possible, since we already do that, and it is more easily understandable by artists/typical people, and also already integrates with our current artist/admin dashboard pages
 - let's use BPS for AB secondary sales royalties so we can defined it as 2.5% (i.e. 250 BPS)
 
I don't have super strong opinions here, other than want to balance internal conventions with backwards-compatibility. Also, we can always update names in follow-on PRs, so nothing is permanent until we deploy.

Reply:
@mchrupcala - how do you feel about merging this as-is, and if we want some further discussion, we can add an issue?

Reply:
per discussion, opened #240 for follow-on discussion of this topic 🚀

Reply:
> merging this as-is

Yes I totally agree, this shouldn't block merging in your PR 🚀 🚀  thanks Ryley!